worlds/stardew_valley/rules.py

```diff
@@ -893,12 +897,14 @@ def set_magic_spell_rules(logic: StardewLogic, multiworld: MultiWorld, player: i
 def set_sve_rules(logic: StardewLogic, multiworld: MultiWorld, player: int, world_options: StardewValleyOptions):
     if ModNames.sve not in world_options.mods:
         return
-    MultiWorldRules.set_rule(multiworld.get_entrance(SVEEntrance.forest_to_junimo, player),
-                             logic.received("Abandoned House Outskirts Clean-up"))
+    MultiWorldRules.set_rule(multiworld.get_entrance(SVEEntrance.forest_to_lost_woods, player),
+                             logic.bundle.can_complete_community_center)
```

Comment:
Forest to Junimo warp requires completion of the community center.  When I initially made the push, it was under the assumption that I could simply open the warp without needing a CC clear.  Sadly the content patcher code that actively injects this event does so only directly after its completion, in a way I am unsure how to patch without a direct dependency.

Thus, I dropped the item, and I put a hard CC requirement for the time being.  There is another way in there through the runes, but otherwise you have to complete the CC.

Reply:
As for the "lost woods" bit, I split Junimo Woods in half just in case for the sake of the nexus warp.  Junimo Woods' top half that you normally enter first is a forest maze, which leads to a clearing where the Junimo are, hence the name.